docker-compose.yml

```diff
@@ -20,8 +20,9 @@ services:
       - db
 
   db:
+    hostname: db
     image: postgres:14-alpine
     ports:
-      - "5432:5432"
```

Comment:
Why do you change the port? 5432 is the standard port for postgres

Reply:
I have PostgreSQL installed on my computer, and I can't connect to db, cos port 5432 default in local db.